services/datamanager/builtin/builtin.go

```diff
@@ -197,6 +197,9 @@ func (b *builtIn) Reconfigure(ctx context.Context, deps resource.Dependencies, c
 		// If this error occurs it's a resource graph error
 		return err
 	}
+	if err := os.MkdirAll(captureConfig.CaptureDir, 0o700); err != nil {
```

Comment:
nit, would `0o600` suffice?

Reply:
probably but we use `0o700` https://github.com/viamrobotics/rdk/blob/33be91481fae0f4247d0fd41c3abfe0a7a86ac68/services/datamanager/builtin/capture/capture.go#L224 in data capture (the existing place where the capture directory gets created) so I kept the same permissions for consistency.